metrotranscom/doorway
# fix: remove contact preferences

This PR addresses #935 


## Description

This PR updates the application flow in Doorway to no longer include the how would you like to be contacted question and  update the household member relation options. Additionally, it removes the work address fields from the application export since it was missed on the last removal. 

## How Can This Be Tested/Reviewed?

This can be tested by going through the online application in different languages/paper application on the partners portal and seeing the changes described above. Additionally, check the applications export and verify that there aren't any references to contact preferences or work addresses.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
